lib/ConnectionContextBase.ts

```diff
@@ -157,6 +162,18 @@ export module ConnectionContextBase {
       operationTimeoutInSeconds: parameters.operationTimeoutInSeconds
     };
 
+    if (parameters.config.webSocket || typeof WebSocket !== 'undefined') {
+      const socket = parameters.config.webSocket || WebSocket;
+      const ws = rhea.websocket_connect(socket);
+      const host = parameters.config.host;
+      const endpoint = parameters.config.webSocketEndpointPath || '';
+
+      // connectionOptions is cast to any to work around type errors.
+      // The errors seem spurious and so are ignored.
```

Comment:
Can we not fix the typings in `https://github.com/amqp/rhea/blob/master/typings/ws.d.ts`?

Reply:
I'm not sure on what or how to fix it, really. Would take some investigation. I think this hack is fine for now?